sourmash_lib/sourmash_args.py

```diff
@@ -6,16 +6,37 @@
 from . import signature
 from .logging import error, notify
 
-def add_moltype_args(parser, default_dna=None):
-    parser.add_argument('--protein', dest='protein', action='store_true')
+
+def add_moltype_args(parser):
+    parser.add_argument('--protein', dest='protein', action='store_true',
+                        help='choose a protein signature (default: False)')
+    parser.add_argument('--no-protein', dest='protein',
+                        action='store_false',
+                        help='do not choose a protein signature')
+    parser.set_defaults(protein=False)
+
+    parser.add_argument('--dna', dest='dna', default=None,
+                        action='store_true',
+                        help='choose a DNA signature (default: True)')
+    parser.add_argument('--no-dna', dest='dna', action='store_false',
+                        help='do not choose a DNA signature')
+    parser.set_defaults(dna=None)
+
+
+def add_construct_moltype_args(parser):
```

Comment:
accidental duplication with `add_construct_moltype_args` and `add_moltype_args`? I can't spot the difference :-/